src/components/Spinner.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div v-if="start">
```

Comment:
Spinner shouldn't have the `start` prop or the `v-if`. We are going to control it in the parent